site/docusaurus.config.js

```diff
@@ -0,0 +1,183 @@
+            require.resolve('@radix-ui/colors/yellow.css'),
+          ],
+        },
+      }),
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      colorMode: {
+        defaultMode: 'dark',
+        disableSwitch: true,
+        respectPrefersColorScheme: false,
+      },
+      docs: {
+        sidebar: { hideable: true, autoCollapseCategories: true },
+      },
+      navbar: {
+        style: 'dark',
+        title: 'Kysely',
+        logo: {
+          alt: 'My Site Logo',
+          src: 'img/logo.svg',
+        },
+        items: [
+          {
+            type: 'doc',
+            docId: 'intro',
+            position: 'left',
+            label: 'Docs',
+          },
+
+          {
+            href: 'https://github.com/koskimas/kysely',
+            label: 'GitHub',
+            position: 'right',
+          },
+          {
+            href: 'https://koskimas.github.io/kysely/',
+            label: 'API docs',
+            position: 'right',
+          },
+        ],
+      },
+      footer: {
+        style: 'dark',
+
+        links: [
+          {
+            title: 'Docs',
+
+            items: [
+              {
+                label: 'Introduction',
+                to: '/docs/intro',
+              },
+              {
+                label: 'Getting started',
+                to: '/docs/installation',
+              },
+              {
+                label: 'Getting started',
+                to: '/docs/installation',
+              },
+              {
+                label: 'Playground',
+                to: '/docs/playground',
+              },
+              {
+                label: 'Migrations',
+                to: '/docs/migrations',
+              },
+              {
+                label: 'Examples',
+                to: '/docs/category/examples',
+              },
+              {
+                label: 'Generating types',
+                to: '/docs/generating-types',
+              },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'Discord',
+                href: 'https://discord.gg/xyBJ3GwvAm',
+              },
+              {
+                label: 'Twitter',
+                href: 'https://twitter.com/kysely_',
+              },
+              // {
+              //   label: 'Stack Overflow',
+              //   href: 'https://stackoverflow.com/questions/tagged/kysely',
+              // },
+            ],
+          },
+          {
+            title: 'Other',
+            items: [
+              {
+                label: 'GitHub',
+                href: 'https://github.com/koskimas/kysely',
+              },
+              {
+                label: 'API docs',
+                href: 'https://koskimas.github.io/kysely/',
+              },
+            ],
+          },
+        ],
+        //copyright: `Copyright © ${new Date().getFullYear()}`,
```

Comment:
```suggestion
        copyright: `Copyright © ${new Date().getFullYear()}`,
```

Reply:
Kysely is MIT licensed, are you sure copyrights to a documentation are relevant?

Reply:
> Kysely is MIT licensed, are you sure copyrights to a documentation are relevant?

good point :)
let's just remove that line then